backend/src/api/auth/auth.controller.js

```diff
@@ -59,3 +66,13 @@ exports.loginUser = async (req, res, next) => {
     token: token,
   });
 };
+
+exports.logoutUser = async (req, res, next) => {
+  const tokenUser = req.get('Authorization').replace('Bearer ', '');
```

Comment:
you could pass token to req object in auth middleware and don't duplicate logic

Reply:
Ok, it was fixed. Now token is passed to request object in auth middleware